importer.py

```diff
@@ -121,6 +157,20 @@ def _fix_library_source_bug(course_xml_dir):
             library_f.write(str(lib_element))
 
 
+def _set_course_dates(course_id):
+    start = _get_course_start_date()
+    end = _get_course_end_date()
+    if not (start and end):
+        return
+
+    print >> sys.stderr, 'setting course start and end dates for ', course_id
+    course = MOD_STORE.get_course(course_id)
+    course.start = start
+    course.end = end
+    course.save()
+    MOD_STORE.update_item(course, course._edited_by)
```

Comment:
Does it persists? I've wasted too much time on this and then I gave up see: https://github.com/appsembler/msft-courses/pull/8